repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/intrafilterlogging/RequestLog.java

```diff
@@ -42,10 +40,10 @@
     String requestBody;
     Map<String, String> headers;
 
-    public RequestLog(HttpServletRequest httpServletRequest, Filter filter) throws IOException {
+    public RequestLog(HttpServletRequest httpServletRequest, String currentFilter) throws IOException {
         preamble = "Intrafilter Request Log";
         timestamp = new DateTime().toString();
-        currentFilter = StringUtils.isEmpty(filter.getId()) ? filter.getName() : filter.getId() + "-" + filter.getName();
+        this.currentFilter = currentFilter;
```

Comment:
This change may make some functional tests fail, but it doesn't seem like anyone was using the id field anyway.